contrib/plugins/funds/funds.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+""" This plugin gives you a nicer overview of the funds that you own.
+
+Instead of calling listfunds and adding all outputs and channels
+this plugin does that for you.
+
+Activate the plugin with: 
+`lightningd --plugin=PATH/TO/LIGHTNING/contrib/plugins/funds/funds.py`
+
+Call the plugin with: 
+`lightning-cli funds`
+
+
+Author: Rene Pickhardt (https://ln.rene-pickhardt.de)
+Development of the plugin was sponsored by fulmo.org
+"""
+
+import json
+
+from lightning.lightning import LightningRpc
+from lightning.plugin import Plugin
+from os.path import join
+
+rpc_interface = None
+plugin = Plugin(autopatch=True)
+
+
+@plugin.method("funds")
+def funds(plugin, unit="s"):
+    """Lists the total funds the lightning node owns off- and onchain in {unit}.
+
+{unit} can take the following values:
+s, satoshi, satoshis to depict satoshis
+b, bit, bits to depict bits
+m, milli, btc to depict milliBitcoin
+B, bitcoin, btc to depict Bitcoins
+
+When not using Satoshis (default) the comma values are rounded off."""
+
+    switcher = {
```

Comment:
Minor optimization and readability improvement: these `dict`s can be instantiated globally, since they don't change at all, and it'd safe us from instantiating them on every call.

Furthermore since they serve different purposes they ought to have different names as well. Maybe call this `unit_aliases` and the one below `unit_divisors`?

Reply:
will fix